src/Microsoft.OData.Core/ODataMessageWriterSettings.cs

```diff
@@ -391,6 +393,18 @@ internal SelectedPropertiesNode SelectedProperties
         /// </summary>
         public bool EnableWritingKeyAsSegment { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether the XML representation of the "Scale" attribute 
+        /// should use lowercase for the value "Variable".
+        /// </summary>
+        public bool UseLowercaseScaleVariable { get; set; }

```

Comment:
Is there any 'variable' in OData spec? any other 'string' value used in OData spec?
If yes, I'd suggest to use one config for them? 
"EnableLowerCaseString" ?

Reply:
I think it's just for the `Scale` and `SRID`